index.html

```diff
@@ -39,7 +39,10 @@
 
     <mgt-login></mgt-login>
     <mgt-person person-query="me" view="twoLines" person-card="hover" show-presence></mgt-person>
+    <br />
```

Comment:
Since there is a story in storybook for this, we don't need to modify this file, please undo the changes here.